app/views/layouts/application.html.erb

```diff
@@ -28,6 +28,15 @@
 
 <body id="<%= @section %>">
 
+  <div class="inner" id="#promo">
```

Comment:
This should be just `id="promo"`; the `#` in the CSS denotes an id, and isn't part of the name.

However, the output doesn't look great. I think the more modern styles we did for the SFC banner last year look nicer, so I'm re-working this PR based on that.